packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -182,7 +182,10 @@ export class ForkChoice implements IForkChoice {
    */
   updateHead(): IBlockSummary {
     // balances is not changed but votes are changed
-    let timer;
+
+    let timer, prevHead, newHead;
```

Comment:
Not a fan of scoped variables here, is really necessary to put the logic in the finally block?

Reply:
> > > screenshot of the new fork-choice section in lodestar grafana dashboard
> > 
> > 
> > Nice!
> > 
> > * Can you select seconds as Y axis unit for the first chart?
> > * Also can you move this row down since it's not as important of the first rows? I would put it after Regen Queuin
> 
> ok (for Y Axis). in the final checkin (after the screenshot), the section is already on the bottom.

ummm, Y Axis is already seconds for the first chart.

Reply:
reason is to let the code do its thing and do do metrics in finally block irrespective if there was an error thrown inside  this or any other function call made by this code. newHead will only be successfully assigned to this.head at the last moment of return. in my opinion it makes for simpler handling.